docker/run

```diff
@@ -34,6 +34,10 @@ def run_dev_environment(command, ros="melodic", gazebo="9"):
     docker_args.append("--volume {}:/{}/src".format(os.path.dirname(ut.get_repo_root()), WS_NAME))
     docker_args.append("--volume {}/ws/build/:/{}/build/".format(temp_volume, WS_NAME))
     docker_args.append("--volume {}/ws/devel/:/{}/devel/".format(temp_volume, WS_NAME))
+    # VSCode needs HOME to be defined in order to work in the container
+    docker_args.append("-e HOME=/home/create")
+    # Copy .bashrc
+    docker_args.append("--volume ~/.bashrc:/home/create/.bashrc")
```

Comment:
Is this needed only to source the workspace? If so, it will fail locally. (I don't have ROS Melodic in my local computer)

Reply:
That's basically to have your host configuration inside the docker. We could add a parameter to select if you want this or not (Or just remove it if you don't want it) but it's more comfortable to have the same shell style when you go into the docker.
Also, I highly recommend you to tune your `.bashrc` with the most common ros version you use (as for example melodic and kinetic) with something like this:
```
ROS_PATH="/opt/ros"
ROS_M_VERSION="melodic"
ROS_K_VERSION="kinetic"
V="$ROS_M_VERSION"

if [ ! -d "$ROS_PATH/$ROS_M_VERSION" ]  && [ -d "$ROS_PATH/$ROS_K_VERSION" ]; then
	V="$ROS_K_VERSION"
fi
source "$ROS_PATH/$V/setup.bash"
```
It basically sources kinetic if finds it, or melodic by default.